content/docs/start/data-pipelines.md

```diff
@@ -105,18 +105,18 @@ prepare:
   two files in it. This is how the <abbr>workspace</abbr> should look like now:
 
   ```git
-   .
-   ├── data
-   │   ├── data.xml
-   │   ├── data.xml.dvc
+     .
+     ├── data
+     │   ├── data.xml
+     │   ├── data.xml.dvc
   +│   └── prepared
   +│       ├── test.tsv
```

Comment:
Here the workaround so it looks correct really makes the code looks broken. And the formatter doesn't catch this one. So I wonder if it's best to close this PR and focus on fixing #1509 instead.

Reply:
to be honest looks like a minor problem, I would wait on the engine to fix this, for now broken it's fine to unindent code blocks